workloads/network-perf/run_network_compare.sh

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env bash
+set -x
+
+datasource="elasticsearch"
+tool="uperf"
+function="compare"
+base_uuid=$1
+if [ "$base_uuid" == "" ]; then
+  echo "Unable to execute compare - no base UUID passed"
+  exit 1
+fi
+compare_uuid=$2
+if [ "$compare_uuid" == "" ]; then
+  echo "Unable to execute compare - no compare UUID passed"
+  exit 1
+fi
+es=${ES_SERVER}
+if [ "$es" == "" ]; then
+  echo "Unable to execute compare - no ES passed"
+  exit 1
+fi
+
+git clone https://github.com/cloud-bulldozer/touchstone
+cd touchstone
+python -m venv ./compare
+source ./compare/bin/activate
+pip install pyyaml
```

Comment:
can this be updated to `pip install -r requirements.txt` ?

Reply:
yup!